edit/location.go

```diff
@@ -31,13 +31,14 @@ func init() {
 var PinnedScore = math.Inf(1)
 
 type location struct {
-	home     string // The home directory; leave empty if unknown.
-	all      []storedefs.Dir
-	filtered []storedefs.Dir
+	home            string // The home directory; leave empty if unknown.
+	all             []storedefs.Dir
+	filtered        []storedefs.Dir
+	caseInsensitive bool
```

Comment:
Use ignoreCase in accordance with the matcher flag `&ignore-case`. Same for below.